sdcm/cluster.py

```diff
@@ -920,11 +920,13 @@ def get_seed_nodes_private_ips(self):
             with open(yaml_dst_path, 'r') as yaml_stream:
                 conf_dict = yaml.load(yaml_stream)
                 try:
-                    self.seed_nodes_private_ips = conf_dict['seed_provider'][
-                        0]['parameters'][0]['seeds'].split(',')
-                except:
-                    raise ValueError('Unexpected scylla.yaml '
-                                     'contents:\n%s' % yaml_stream.read())
+                    self.seed_nodes_private_ips = conf_dict['seed_provider'][0]['parameters'][0]['seeds'].split(',')
+                except Exception, details:
```

Comment:
+                self.log.debug('dict of loaded yaml: %s' % str(conf_dict))

Output the parsed dict here, it's helpful. We can copy it to python cmdline and debug the code.